config/kustomize/watch-all/kustomization.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
For the watch-all case
the [internal/deploy/kustomize/daily/overlays/watch-all-namespaces/wlo-all-namespaces.yaml](https://github.com/WASdev/websphere-liberty-operator/blob/main/internal/deploy/kustomize/daily/overlays/watch-all-namespaces/wlo-all-namespaces.yaml) appears to be missing the `namespace: websphere-liberty` flag that should be included similar to 1.2.0's [deploy/releases/1.2.0/kustomize/overlays/watch-all-namespaces/wlo-all-namespaces.yaml](https://github.com/WASdev/websphere-liberty-operator/blob/main/deploy/releases/1.2.0/kustomize/overlays/watch-all-namespaces/wlo-all-namespaces.yaml).